portfolio/src/main/java/com/google/sps/servlets/CalendarServlet.java

```diff
@@ -0,0 +1,33 @@
+package com.google.sps.servlets;
+
+import com.google.gson.Gson;
+import com.google.sps.data.CalendarEvent;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Servlet that handles calendar event creation. */
+@WebServlet("/calendarServlet")
+public class CalendarServlet extends HttpServlet {
+
+  public List<CalendarEvent> eventList;
```

Comment:
Consider simplifying the variable name to "events"

Reply:
Updated.